core/test/martian/core_test.cljc

```diff
@@ -311,9 +311,27 @@
            (martian/request-for m :create-blob {::martian/body body})
            (martian/request-for m :create-blob {:blob body})))))
 
+(deftest missing-route-test
+  (let [m (martian/bootstrap "https://camels.org"
+                             [{:route-name :create-camel
```

Comment:
Could slim down this test by just giving an empty array for the routes

Reply:
Done